### PR TITLE
Add mootable-hashmap w/ npm auto-update

### LINKED ISSUE #729: Add: Mootable Hashmap
### Library name

mootable-hashmap

### Library license

https://github.com/mootable/hashmap/blob/master/LICENSE

### Library source

https://github.com/mootable/hashmap/

### PR DESCRIPTION
**Pull request by jackmoxley**
Add new library @mootable/hashmap w/ npm auto-update
Resolves #729

### CODE REVIEW

File: packages/m/mootable-hashmap.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "url": "git+https://github.com/mootable/hashmap.git"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "@mootable/hashmap",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*.@(js|map)"
```
Comment (MattIPv4) [inline packages/m/mootable-hashmap.json:24]:
```suggestion
          "*.@(js|map)"
```
It looks like all the files are just in this directory, so let's avoid a globstar when we can.

File: packages/m/mootable-hashmap.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "description": "Full featured, fast, HAMT based HashMaps",
+  "filename": "index.js",
```
Comment (MattIPv4) [inline packages/m/mootable-hashmap.json:3]:
```suggestion
  "filename": "hashmap.min.js",
```
It looks like the files are called hashmap, not index :)

**Review by MattIPv4 — CHANGES_REQUESTED** [review]

File: packages/m/mootable-hashmap.json (hunk 2 above)
Comment (jackmoxley) [reply packages/m/mootable-hashmap.json:3]: sure is hashmap.min the correct one or just hashmap, I provide both? For now I will make it just hashmap.

**Comment by jackmoxley** [comment]
Changes made :-)

**Review by MattIPv4 — APPROVED** [review]
Lgtm, ty!

**Pull request merged**